vasttools/tools.py

```diff
@@ -227,16 +231,30 @@ def _create_fields_df(epoch_num: str,
 
     Raises:
         Exception: Path does not exist.
+        Exception: No data available for epoch.
     """
     field_columns = ['FIELD_NAME', 'SBID', 'SCAN_START', 'SCAN_LEN']
 
     vast_db = Path(db_path)
     if not vast_db.exists():
         raise Exception("{} does not exist!".format(vast_db))
 
-    if type(epoch_num) is int:
-        epoch_num = str(epoch_num)
-    epoch = vast_db / 'epoch_{}'.format(epoch_num.replace('x', ''))
+    if type(epoch_num) is str:
+        epoch_num = int(epoch_num.replace('x', ''))  # .lstrip('0')
```

Comment:
```suggestion
        epoch_num = int(epoch_num.replace('x', ''))
```
Is that ok to be removed?